### ISSUE TITLE
Support patching .aea otas

### ISSUE THREAD

**Issue by mduran80** [blacktop/ipsw#566]
### Is your feature request related to a problem? Please describe.

`ipsw ota patch` currently expects a zip file. New otas encrypted as .aea cannot be used with the patch command after decryption. 

### Describe the solution you'd like

Use `ota patch` transparently

### Describe alternatives you've considered

I have tried a workaround to create the cryptexes but unfortunately didn't work as there's no BuildManifest in the archive contents.
1. Decrypt .aea into a temp aar file `aea decrypt -i .... -o <temp_aar>`
2. Patch the temp aar file into a temp folder using `aa patch -i <temp_aar> -d <temp_folder>`
3. Zip the <temp_folder> into <temp_zip>
4. Run ipsw ota patch rsr <temp_zip> 

```
  ⨯ failed to get OTA folder: no BuildManifest.plist found
```

### Search

- [X] I did search for other open and closed issues before opening this

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blacktop** [comment]
Has there been an RSR since the arrival of AEA? I think RSRs are dead

**Comment by mduran80** [comment]
No, not RSR, just the iOS 18 OTA for iPhone17,1 et al. I have been using the cmd "rsr" to obtain the cryptexes from full OTAs without prerequisites. I don't usually work with deltas and it was the only way I could make it work. Is there another alternative to extract the cryptexes?

**Comment by mduran80** [comment]
I finally found a way although a bit hacky... As ipsw was demanding a BuildManifest.plist, I just copied it from `<patched_tmp>/boot/BuildManifest.plist` to the root of the folder before zipping. Afterwards, `ipsw ota patch` kindly created the cryptex dmgs.

**Comment by blacktop** [comment]
you want the full DMGs right not just files from the DMGs?

**Comment by dff-mdg** [comment]
I don't know how I missed your reply... sorry for the late response. Yes, full DMGs

**Comment by mduran80** [comment]
yeah, full DMGs

**Comment by blacktop** [comment]
added `ipsw ota extract --cryptex system` will create `cryptex-system.dmg` etc
